src/logtab.cpp

```diff
@@ -145,9 +177,6 @@ void LogTab::InitOneRowMenu()
     QAction *findNextSelectedType = new QAction(QIcon(GetThemedIcon(":/ctx-down.png")), "Find next event of this type", this);
     connect(findNextSelectedType, &QAction::triggered, this, &LogTab::RowFindNextSelectedType);
 
-    QAction *highlightSelectedType = new QAction(QIcon(GetThemedIcon(":/ctx-highlight.png")), "Highlight all events of this type", this);
```

Comment:
This action has been at the top level of the context menu for a while.
Would it make sense to keep it as is and simply add a menu? With duplication of "this key" or without it.

Reply:
Adjusted the commits such that the 'Highlight all events of this type' is still available in the top-level menu